### PR TITLE
Update sys-dm-database-backups-azure-sql-database.md

### PR DESCRIPTION
**Pull request by sqlworldwide**
Friendlier resultset that is easy to consume

### CODE REVIEW

**Comment by PRMerger8** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by AnnaMHuff** [comment]
@SudhirRaparla - Could you review this proposed update to your article and enter `#sign-off` in a comment if it's approved for merge?

Thanks!

#label:"aq-pr-triaged"

**Review by SudhirRaparla — CHANGES_REQUESTED** [review]
Hi, Thank you for your feedback. Scope of sys.dm_database_backups is a database i.e., you have to be connected to a database to view its backups. So database name might be redundant, unless you see other use cases where db name is useful. For the same reason we can remove db.name <> 'master' filter. Using case statement to explicitly call out the backup type and retention type definitely makes it more user friendly.

File: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md [hunk 1]
```diff
@@ -56,3 +56,49 @@ SELECT *
 FROM sys.dm_database_backups     
 ORDER BY backup_finish_date DESC;  
 ```  
+A friendlier resultset by joining with sys.databases and using CASE statement. Run this query in master database to get backup history for all databases in the Azure SQL Server.
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md:59]:
```suggestion
You can get a friendlier resultset by joining to `sys.databases` and using a `CASE` statement. Run this query in the `master` database to get backup history for all databases in the Azure SQL Database server.
```

File: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md [hunk 2]
```diff
@@ -56,3 +56,49 @@ SELECT *
...
+ 
+ ```
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md:61]:
```suggestion
 ```sql
```

File: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md [hunk 3]
```diff
@@ -56,3 +56,49 @@ SELECT *
...
+ SELECT
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md:62]: Here is the same query using our formatting:

```
SELECT db.name
	, backup_start_date
	, backup_finish_date
	, CASE backup_type
		WHEN 'D' THEN 'Full'
		WHEN 'I' THEN 'Differential'
		WHEN 'L' THEN 'Transaction Log'
		END AS BackupType
	, CASE in_retention
		WHEN 1 THEN 'In Retention'
		WHEN 0 THEN 'Out of Retention'
		END AS is_Bakcup_Available
FROM sys.dm_database_backups AS ddb
INNER JOIN sys.databases AS db
	ON ddb.physical_database_name = db.physical_database_name
ORDER BY backup_start_date DESC;
```

File: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md [hunk 4]
```diff
@@ -56,3 +56,49 @@ SELECT *
...
+--excluding master database
+WHERE db.name <> 'master'
+*/
+ORDER BY backup_start_date DESC
+```
+
+Run the below query in the user database context to get backup history for single database.
+
+```
+SELECT
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md:88]: Here is the same query using our formatting:

```
SELECT backup_start_date
	, backup_finish_date
	, CASE backup_type
		WHEN 'D' THEN 'Full'
		WHEN 'I' THEN 'Differential'
		WHEN 'L' THEN 'Transaction Log'
		END AS BackupType
	, CASE in_retention
		WHEN 1 THEN 'In Retention'
		WHEN 0 THEN 'Out of Retention'
		END AS is_Bakcup_Available
FROM sys.dm_database_backups AS ddb
INNER JOIN sys.databases AS db
	ON ddb.physical_database_name = db.physical_database_name
ORDER BY backup_start_date DESC;
```

File: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md [hunk 5]
```diff
@@ -56,3 +56,49 @@ SELECT *
...
+/*
+--excluding master database
+WHERE db.name <> 'master'
+*/
+ORDER BY backup_start_date DESC
+```
+
+Run the below query in the user database context to get backup history for single database.
+
+```
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md:87]:
```suggestion
```sql
```

**Comment by rwestMSFT** [comment]
@sqlworldwide Thank you for your contribution! This is fantastic. I've made suggestions for formatting.

File: docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md [hunk 6]
```diff
@@ -56,3 +56,49 @@ SELECT *
...
+JOIN sys.databases AS db
+ON ddb.physical_database_name = db.physical_database_name
+/*
+--excluding master database
+WHERE db.name <> 'master'
+*/
+ORDER BY backup_start_date DESC
+```
+
+Run the below query in the user database context to get backup history for single database.
```
Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-database-backups-azure-sql-database.md:85]:
```suggestion
Run the below query in the user database context to get backup history for a single database.
```

**Comment by SudhirRaparla** [comment]
Thank you @rwestMSFT for suggesting the changes. @sqlworldwide sorry for the delayed response. Can you please include Randolph's changes and we should be good to go.

**Comment by PRMerger8** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by sqlworldwide — COMMENTED** [review]
making changes recommended by Randolph West.

**Comment by PRMerger-2** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by sqlworldwide** [comment]
@rwestMSFT thank you for your suggestions. I did not get Microsoft's formatting right (especially with space counts). Feel free to change with the standard.

**Comment by rwestMSFT** [comment]
@SudhirRaparla I reached out to @sqlworldwide privately and we’re going to do a PR on his local fork and then push here again. Once that’s done then we can sign it off.

**Comment by PRMerger19** [comment]
@sqlworldwide : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rwestMSFT** [comment]
#sign-off

**Comment by PRMerger6** [comment]
@rwestMSFT: I'm sorry - only the author of this article, @SudhirRaparla, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by rwestMSFT** [comment]
@SudhirRaparla, you can sign off at any time now. @sqlworldwide, we'll merge this in when the content freeze is lifted.

**Comment by SudhirRaparla** [comment]
#Sign-off

**Comment by SudhirRaparla** [comment]
Thank you @rwestMSFT and @sqlworldwide for your contribution.

**Pull request merged**